packages/product-json-to-csv/src/map-product-data.js

```diff
@@ -7,10 +7,13 @@ import type {
   Variant,
   Category,
   Image,
+  Price,
+  Attribute,
+  ProductType,
 } from 'types/product'
 
 import { flatten } from 'flat'
-import { isEmpty, reduce, pickBy, isNil } from 'lodash'
+import _ from 'lodash'
```

Comment:
if I remember correctly, we use `rollup`, and that comes with tree shaking to keep the built bundle small.
And this import beats the purpose of that.

I highly advise that you rely on own implementation of FP which would accomplish a lot of things already.

Reply:
Changed here https://github.com/commercetools/nodejs/pull/680/commits/4e06beed22d16ca96402a4d8148c25e1b0d702cb#diff-9435e999bde99eccdb8481181b8252b8R16

Some functions in Lodash are faster than native ones. Also, it has many optimizations and things which native functions can't do so I would not be afraid of using it more..